packages/error-boundary-fallback/README.md

```diff
@@ -0,0 +1,24 @@
+# Errorboundary Fallback
+
+This package provides a fallback component which you can use as an error boundary fallback.
+
+## Installation
+
+```node
+yarn add @opensrp/error-boundary-fallback
+```
+
+#### Code example
+
+```javascript
+import * as Sentry from '@sentry/react';
```

Comment:
Seems like this import has not been used anywhere in the example code. Could we remove it?